packages/cli/commands/config/list.js

```diff
@@ -0,0 +1,63 @@
+const { logger } = require('@hubspot/cli-lib/logger');
+const { getConfig } = require('@hubspot/cli-lib/lib/config');
+const {
+  loadConfig,
+  validateConfig,
+  checkAndWarnGitInclusion,
+} = require('@hubspot/cli-lib');
+
+const {
+  addConfigOptions,
+  addAccountOptions,
+  addUseEnvironmentOptions,
+  getAccountId,
+  setLogLevel,
+} = require('../../lib/commonOpts');
+const { trackCommandUsage } = require('../../lib/usageTracking');
+const { logDebugInfo } = require('../../lib/debugInfo');
+const { validateAccount } = require('../../lib/validation');
+const { getTableContents, getTableHeader } = require('../../lib/table');
+
+const loadAndValidateOptions = async options => {
+  setLogLevel(options);
+  logDebugInfo(options);
+  const { config: configPath } = options;
+  loadConfig(configPath, options);
+  checkAndWarnGitInclusion();
+
+  if (!(validateConfig() && (await validateAccount(options)))) {
+    process.exit(1);
+  }
+};
+
+exports.command = 'list';
+exports.describe = 'List names of portals defined in config';
+
+exports.handler = async options => {
+  loadAndValidateOptions(options);
+
+  const accountId = getAccountId(options);
+
+  trackCommandUsage('functions-list', {}, accountId);
+
+  const config = getConfig();
+  const portalData = config.portals.reduce((acc, portal) => {
+    acc.push([portal.name, portal.portalId, portal.authType]);
+
+    return acc;
+  }, []);
+  portalData.unshift(getTableHeader(['Name', 'Portal ID', 'Auth Type']));
+
+  logger.log(getTableContents(portalData));
+  logger.log('Default Portal: ', config.defaultPortal);
```

Comment:
I'm not sure if this is the best place for this, but it would be helpful to print out the path of the config file, which might be helpful for folks to know which config the CLI is actually loading

Reply:
Good call, I updated the output from this command to include the config path and made the accounts list a bit nicer too
![image](https://user-images.githubusercontent.com/6472448/116290868-f04a4e80-a761-11eb-8ee4-fa03cc17789f.png)